src/index.js

```diff
@@ -1,43 +1,63 @@
 import './style.css';
+import loadList from './module/loadList.js';
+import storeArray from './module/storeArray.js';
 
-const list = [
-  {
-    description: 'watching tutorial',
-    completed: false,
-    index: 0,
-  },
-
-  {
-    description: 'wash the dishes',
-    completed: false,
-    index: 1,
-  },
+const list = [];
+const mainInput = document.querySelector('.main-input');
+const bigCont = document.querySelector('.container');
+const objCreater = () => {
+  const listObj = {};
+  listObj.description = mainInput.value;
+  listObj.completed = false;
+  listObj.index = list.length;
+  mainInput.value = '';
+  return listObj;
+};
 
-  {
-    description: 'buy groceries',
-    completed: false,
-    index: 2,
-  },
+bigCont.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    if (mainInput.value !== '') {
+      const objects = objCreater();
+      loadList(objects);
+      list.push(objects);
+      list.sort((a, b) => a.index - b.index);
+      storeArray(list);
+      e.preventDefault();
+    }
+  }
+});
```

Comment:
- [ ] Kindly consider when adding a todo and after refreshing, everything has vanished and your data is saved in localStorage and get from it.
- [ ] Also consider when edit todo is saved in local storage and displayed on the screen.
- [ ] Also consider when deleting a todo to update the index for every todo in the list